article/_posts/2019-07-11-Towards-Universal-Object-Detection.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: review
+title: Towards Universal Object Detection by Domain Attention
+tags: deep-learning CNN domain-adaptation detection medical essentials
+cite:
+    authors: "Xudong Wang, Zhaowei Cai, Dashan Gao, Nuno Vasconcelos"
+    title:   "Towards Universal Object Detection by Domain Attention"
+    venue:   "CVPR 2019"
+pdf: "http://openaccess.thecvf.com/content_CVPR_2019/papers/Wang_Towards_Universal_Object_Detection_by_Domain_Attention_CVPR_2019_paper.pdf"
+---
+
+Code available [here](https://github.com/rbgirshick/py-faster-rcnn)
+
+# Introduction
+
+![](/article/images/UODB/benchmark.png)
+
+In this paper, they develop a universal object detection system that is capable of working on various image domains, from human faces and traffic signs to medical CT images.
+
+The proposed domain attention modules:
+![](/article/images/UODB/propose_module.png)
+
+Altogether, UODB(Universal Object Detection Benchmark) covers a wide range of variations in a category, camera view, image style, etc, and thus establishes a good suite for the evaluation of universal/multi-domain object detection.
+
+# Universal Object detection
+
+The detectors of the adaptive multi-domain detector require prior knowledge of the domain of interest.
+This is undesirable for autonomous systems, like robots or self-driving cars, where determining the domain is part of the problem to solve.
+
+The design of universal detectors (show at the previous figure c) is the simplest solution to share a single detector by all tasks.
+But even for this detector, the output layer has to be task specific, by definition of the detection problem.
+They had found that there is also a benefit in using task-specific RPN layers.
+Beyond the detector, the fully shared detector is the most efficient of the other detectors considered in this work, as it has no domain-specific parameters.
```

Comment:
*Beyond the detector, the fully shared detector (...)*

Looks like a contradiction, or as if two detectors existed. I think this deserves some clarification.

Reply:
Addressed in bc38d30.